src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -936,6 +936,17 @@
     <Field type="ThreadState" name="state" label="Thread State" />
   </Event>
 
+  <Event name="CPUTimeExecutionSample" category="Java Virtual Machine, Profiling" label="CPU Time Method Profiling Sample" description="Snapshot of a threads state from the CPU time sampler"
```

Comment:
Maybe we should leave out "Execution" and only have CPUTimeSample?

Reply:
Sounds like a good idea, because it combines both `ExecutionSample` and `NativeMethodSample`.

Reply:
I renamed it